videointelligence/tests/unit/gapic/v1/test_video_intelligence_service_client_v1.py

```diff
@@ -83,17 +83,17 @@ def test_annotate_video(self):
             client = videointelligence_v1.VideoIntelligenceServiceClient()
 
         # Setup Request
-        input_uri = "gs://cloud-samples-data/video/cat.mp4"
         features_element = enums.Feature.LABEL_DETECTION
         features = [features_element]
+        input_uri = "gs://cloud-samples-data/video/cat.mp4"
 
-        response = client.annotate_video(input_uri=input_uri, features=features)
+        response = client.annotate_video(features, input_uri=input_uri)
         result = response.result()
         assert expected_response == result
 
         assert len(channel.requests) == 1
         expected_request = video_intelligence_pb2.AnnotateVideoRequest(
-            input_uri=input_uri, features=features
```

Comment:
Looks like the removal of `input_uri` from the expected request is making the test fail.

```
=================================== FAILURES ===================================
____________ TestVideoIntelligenceServiceClient.test_annotate_video ____________

self = <test_video_intelligence_service_client_v1.TestVideoIntelligenceServiceClient object at 0x7f9dab53f490>

    def test_annotate_video(self):
        # Setup Expected Response
        expected_response = {}
        expected_response = video_intelligence_pb2.AnnotateVideoResponse(
            **expected_response
        )
        operation = operations_pb2.Operation(
            name="operations/test_annotate_video", done=True
        )
        operation.response.Pack(expected_response)

        # Mock the API response
        channel = ChannelStub(responses=[operation])
        patch = mock.patch("google.api_core.grpc_helpers.create_channel")
        with patch as create_channel:
            create_channel.return_value = channel
            client = videointelligence_v1.VideoIntelligenceServiceClient()

        # Setup Request
        features_element = enums.Feature.LABEL_DETECTION
        features = [features_element]
        input_uri = "gs://cloud-samples-data/video/cat.mp4"

        response = client.annotate_video(features, input_uri=input_uri)
        result = response.result()
        assert expected_response == result

        assert len(channel.requests) == 1
        expected_request = video_intelligence_pb2.AnnotateVideoRequest(
            features=features
        )
        actual_request = channel.requests[0][1]
>       assert expected_request == actual_request
E       assert features: LABEL_DETECTION\n == input_uri: "gs://cloud-samples-data/video/cat.mp4"\nfeatures: LABEL_DETECTION\n

```

Reply:
Thanks. I'm following up with the ML APIs team to see who can look into this.

Reply:
Is there an issue tracking the generator bug?

Reply:
Filed https://github.com/googleapis/gapic-generator/issues/3004

Reply:
Generator bug has been resolved. I will regen this next week to pick up the fix.